audio-worklet/src/noisegate-script-processor.js

```diff
@@ -40,39 +52,39 @@ class NoiseGate {
 
     this.context_ = context;
     const bufferSize = options.bufferSize || 0;
-    // The default threshold is 140 decibels (loudest sound a human can hear
-    // without pain) so that the noise gate does not mute signals by default.
-    this.threshold = options.threshold || 140;
+    this.threshold = options.threshold || 0;
     this.attack = options.attack || 0;
     this.release = options.release || 0;
+    
+    // The time constant of the filter has been set experimentally to minimize
+    // delay while still adequately suppressing high frequency oscillation.
```

Comment:
Nit: Leave out things like "minimize" and "adequately" because you haven't quantified these in any way.  Something like "to balance roughly delay for high frequency suppression".  Or something.